sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/Jsr305AnnotationsCheck.java

```diff
@@ -532,11 +532,78 @@ private AbstractJsr305Handler handleDefinition(final DetailAST ast) {
     private static void addNextNullnessAnnotation(final Set<NullnessAnnotation> result,
             DetailAST ast) {
         final DetailAST identifier = ast.findFirstToken(TokenTypes.IDENT);
-        final String annotationName = identifier.getText();
-        final NullnessAnnotation annotation = STRING2ANNOTATION.get(annotationName);
-        if (annotation != null) {
-            result.add(annotation);
+        if (identifier != null) {
+            final String annotationName = identifier.getText();
+            final NullnessAnnotation annotation = STRING2ANNOTATION.get(annotationName);
+            if (annotation != null) {
+                result.add(annotation);
+            }
+        }
+    }
+
+    /**
+     * Is the current symbol a primitive type.
+     * @param ast
+     *        the ast
+     * @return true if yes
+     */
+    private static boolean isPrimitiveType(DetailAST ast) {
```

Comment:
Why did these methods move from `AbstractJsr305Handler`? I don't see them needing anything from the class they were moved to.
Why not just move these methods over the locations they were originally?

Reply:
I did that when implementing the (subsequently removed) reflection-based unit test and left them there afterwards because I usually like static stuff better than non-static stuff. I'd like to leave it as it is now, but if you request it I can move it back.